lib/transport/selenium-webdriver/index.js

```diff
@@ -163,7 +164,7 @@ class SeleniumWebdriver extends WebdriverTransport {
               try {
                 body = JSON.stringify(data);
               } catch (err) {
-
+                // 
```

Comment:
```suggestion

```

Reply:
In order to avoid warning of `no-empty` rule in eslint I added that.